modules/ROOT/pages/design-trusted-domains.adoc

```diff
@@ -0,0 +1,20 @@
+= Allowlisting Trusted Domains
+
+Due to security reasons, to make an HTTP request to an external domain using the Anypoint proxy, you need to register it as a trusted domain in Anypoint Platform.
```

Comment:
I'm not sure how aware are our customers of the usage of the proxy. I mean, calls to the proxy happen under the hood, i.e. customers are typically unaware of it. We should probably state that every request to external resources goes through the API Console Proxy, and, for security reasons, these domains need to be previously registered as trusted